app/lib/features/pins/models/create_pin_state/pin_attachment_model.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:acter/common/models/types.dart';
+
+class PinAttachment {
```

Comment:
does that have to be bound to pin? shouldn't this be in the `attachment`s-features and something that can be reused for different cases?

Reply:
Currently this is bound to Pin but yes in later PR it will be bound to attachment as common picker. Don't wanted to do attachment related additional changes in this PR.